build_local_windows.ps1

```diff
@@ -26,22 +26,12 @@ $myhome = $HOME.replace("\", "/")
 # release to a folder in the current user's home directory.
 $config_yaml =
 "storage: `
-    azure: `
-     kind: azure_block_blob `
-     account_name: $env:AZURE_STORAGE_ACCOUNT `
-     account_key: $env:AZURE_STORAGE_ACCESS_KEY `
-     container: dcos `
-     download_url: https://dcos.azureedge.net/dcos/dcos-windows/ `
-    aws: `
-        kind: aws_s3 `
-        access_key_id: $env:AWS_ACCESS_KEY_ID `
-        secret_access_key: $env:AWS_SECRET_ACCESS_KEY `
-        bucket: downloads.dcos.io `
-        object_prefix: dcos/dcos-windows `
-        download_url: https://downloads.dcos.io/dcos/dcos-windows/
```

Comment:
this cannot be deleted otherwise the packages aren't uploaded to s3 and azure

Reply:
there are powershell cmd-lets below in the script which uploads to S3 single artifact - `dcos_generate_config_win.sh`
See this , pls:
```
## Set and Read AWS Credentials:
Set-AWSCredential -AccessKey $env:AWS_ACCESS_KEY_ID -SecretKey $env:AWS_SECRET_ACCESS_KEY -StoreAs aws_s3_windows
Set-AWSCredential -ProfileName aws_s3_windows
## Upload Tar Ball to dcos.download.io
Write-S3Object -BucketName "s3-us-west-2" -Key "downloads.dcos.io\dcos\windows\$env:BRANCH\commit\$env:COMMIT\dcos_generate_config_win.sh" -File "$HOME\dcos-artifacts\windows\dcos_generate_config_win.sh" -CannedACLName public-read
```

Reply:
yes, with this instruction the one installer file for windows is uploaded. 
Storage providers cannot be removed because then the individual upload of each component is completely removed. This feature is important for caching builds and cannot be removed. So please reset the storage provider configuration to the original settings.